src/output-json-tls.c

```diff
@@ -337,6 +455,18 @@ void JsonTlsLogJSONExtended(json_t *tjs, SSLState * state)
 
     /* tls notafter */
     JsonTlsLogNotAfter(tjs, state);
+
+    /* tls subject public key algorithm */
+    JsonTlsLogSubjectPublicKeyAlgo(tjs, state);
+
+    /* tls certificate signature algorithm */
+    JsonTlsLogCertificateSignatureAlgo(tjs, state);
+
+    /* tls signature algorithm */
+    JsonTlsLogSignatureAlgo(tjs, state);
+
+    /* tls extensions */
+    JsonTlsLogExtensions(tjs, state);
```

Comment:
In my opinion, this is probably not something everybody would want to log, so it should be added to custom logging instead. Should also change name to something other than 'extensions'.